keps/NNNN-kep-template/README.md

```diff
@@ -420,6 +428,9 @@ automations, so be extremely careful here.
 Describe the consequences on existing workloads (e.g., if this is a runtime
 feature, can it break the existing applications?).
 
+Feature gates are typically disabled by setting the flag and restarting the
+component. No other changes should be necessary to disable the feature.
```

Comment:
```suggestion
Feature gates are typically disabled by setting the flag to `false` and restarting the
```

Reply:
Oops, good catch

Reply:
oh oops the suggestion deleted the second half of that line. let me fix manually